### DISCUSSION TITLE
HyperHDR for HDMI inputs (Apple TV 4k) not working on LG TV webOS

### DISCUSSION THREAD

**Discussion by kira-95** [awawa-dev/HyperHDR#886] [General]
I have Apple TV 4k 3rd gen connected to my rooted LG C1 through the HDMI port 1 (not eARC).

The rooted LG TV has piccap and Hyper HDR 20.0.0.0 installed.

When using the Apple TV 4k, whenever I restart the TV, I can see the capture in the live video in HyperHDR but after a few seconds, it goes blank.

Eveything works fine on the webOS, issues only exist when using the Apple TV.

I set the video format on the Apple TV to be 4k SDR 60.

I see the following error message :

[23:06:42.953][ERR] hyperion-webos ERR capture_acquire_frame: vtCapture_currentCaptureBuffInfo() failed. {}

**Comment by awawa-dev** [comment]
Hi
HyperHDR does not perform image capture on the webos, it is only a receiver. This log is also not the log from our application but from this: https://github.com/webosbrew/hyperion-webos Maybe you need to activate some quirk in piccap..dont know.
